js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/AutomaterializeRunsTable.tsx

```diff
@@ -0,0 +1,113 @@
+import {gql, useQuery} from '@apollo/client';
+import {Body2, Box, Mono, Table, colorTextLighter} from '@dagster-io/ui-components';
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+import {PYTHON_ERROR_FRAGMENT} from '../../app/PythonErrorFragment';
+import {PythonErrorInfo} from '../../app/PythonErrorInfo';
+import {RunStatusTagWithStats} from '../../runs/RunStatusTag';
+import {RUN_TIME_FRAGMENT, RunStateSummary, RunTime, titleForRun} from '../../runs/RunUtils';
+
+import {
+  AutomaterializeRunsQuery,
+  AutomaterializeRunsQueryVariables,
+} from './types/AutomaterializeRunsTable.types';
+
+export const AutomaterializeRunsTable = ({runIds}: {runIds: string[]}) => {
+  const {data, loading, error} = useQuery<
+    AutomaterializeRunsQuery,
+    AutomaterializeRunsQueryVariables
+  >(AUTOMATERIALIZE_RUNS_QUERY, {
+    variables: {
+      filter: {
+        runIds,
+      },
+    },
+    skip: !runIds.length,
+  });
+
+  if (!runIds.length) {
+    return (
+      <Body2 color={colorTextLighter()} style={{paddingBottom: 32}}>
+        None
+      </Body2>
+    );
+  }
+
+  if (error) {
+    return <Body2>An error occurred fetching runs. Check your network status</Body2>;
```

Comment:
Seems more likely to be a GraphQL error than a network error here, right?

Reply:
I'm not sure tbh, I've seen this return an error when it's actually a network error though